selftest/common/vmlinux.h

```diff
@@ -1,6 +1,10 @@
 #ifndef __VMLINUX_H__
 #define __VMLINUX_H__
 
+#ifndef BPF_NO_PRESERVE_ACCESS_INDEX
```

Comment:
@mozillazg does this PR really require all these definitions in vmlinux.h?

Reply:
In the previous commit, I used a complete version of `struct task_struct`, so these definitions are necessary. The latest version has been updated to use a slim version of `struct task_struct`.